dwellingly/package-lock.json

```diff
@@ -1000,6 +1000,119 @@
       "resolved": "https://registry.npmjs.org/@csstools/normalize.css/-/normalize.css-10.1.0.tgz",
       "integrity": "sha512-ij4wRiunFfaJxjB0BdrYHIH8FxBJpOwNPhhAcunlmPdXudL1WQV1qoP9un6JsEBAgQH+7UXyyjh0g7jTxXK6tg=="
     },
+<<<<<<< HEAD
```

Comment:
Looks like there was a merge conflict that got left behind. If you run `npm install`, then commit the changes it makes to this file it should be fixed!